### DISCUSSION TITLE
The webportal only allows to take pictures from front camera (laptop and mobile device)

### DISCUSSION THREAD

**Discussion by raxit65535** [community/community#139777] [GitHub Education]
### Hi everyone,

Hi, I was signing up for Github Education and noticed that in the verification process the portal only allow to use front camera of computing device (computer, phone). Ideally the verification process should contain upload button from which a student can upload pictures of verification proof.

**Comment by jdhonau** [comment]
> ### Hi everyone,
> Hi, I was signing up for Github Education and noticed that in the verification process the portal only allow to use front camera of computing device (computer, phone). Ideally the verification process should contain upload button from which a student can upload pictures of verification proof.

I second this complaint.  For something as high tech as a git platform for software engineers,  you'd think there'd be access to the rear camera and (GASP) the ability to upload a document sent by the school enrollment team directly from ones computer or phone.  I had to print mine out, tape it to a manilla folder so I could hold it steady while upright in front of my phone since it wouldn't switch to rear...then I had to hold the phone backwards while the letter and folder was taped to a whiteboard (I couldn't see what I was doing by taping it to a wall) so I could stand behind this large whiteboard it to see where the front camera was pointing...all while trying not to breathe so my hand didn't shake even the tiniest bit or the words would be blurry since front cameras absolutely suck at text (even iphone 15 pro max).  It was the absolute biggest waste of time hobbled by terrible technology, only to be waiting almost 48 hours and still not receive access to the student benefits even AFTER my approval.  I was a bit concerned by the janktastic nature of the verification upload working that I decided to enroll in the local community college, get accepted, and received my .edu email (since my coding bootcamp doesn't give students email addresses)...in less time than it took to get accepted and access to the education pack, and I enrolled simply because I plan to attend as soon as I'm done with bootcamp, and I hoped that the college issued email would have been the linchpin to gaining access to the valuable tools I've been doing without for months that GitHub offers students.  Hopefully I'll get access to my student pack before I'm set to graduate in a few weeks....It's a good thing I'm following graduation with community college to learn languages not offered by my bootcamp.
